### PR TITLE
feat: question crud

### LINKED ISSUE #287: Add CRUD for question

### PR DESCRIPTION
**Pull request by Guekka**
Fix #287

### CODE REVIEW

**Comment by Apoorva64** [comment]
# ☁️ Nx Cloud Report

> ⚠️ Important: This organization has reached the limit of the FREE Plan and will be disabled at the end of the month. Remote caching and distributed task execution will no longer work. Your CI may stop working.
        

CI is running/has finished running commands for commit 81b82caab5b6b246055845f21d6f7accfb7118ad. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=640ee8fb56da4d6500e89717&branchName=288,287-add-crud-for-question)

---


<details>
<summary>✅ Successfully ran 1 target </summary>

- [`nx affected --target=lint,test,build --parallel=3`](https://cloud.nx.app/runs/GwLq9gi6ke)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: apps/backend/api/src/main.ts [hunk 1]
```diff
@@ -56,6 +57,7 @@ app.get('/health', (req, res) => {
 app.get('/api/tts', ttsRouter);
 app.use('/api/accommodation', accommodationRouter);
 app.use('/api/answer', answerRouter);
+app.use('/api/question', questionRouter);
```
Comment (Apoorva64) [inline apps/backend/api/src/main.ts:60]: Should be in plural

**Review by Apoorva64 — CHANGES_REQUESTED** [review]

**Review by ArnaudDumanois — DISMISSED** [review]
lgtm

**Review by ozeliurs — APPROVED** [review]
LGTM

**Review by Apoorva64 — APPROVED** [review]

**Pull request merged**